moviepy/video/io/ffmpeg_reader.py

```diff
@@ -8,9 +8,6 @@
 import subprocess as sp
 import re
 import warnings
-import logging
-logging.captureWarnings(True)
```

Comment:
This would lead to the warning not being shown. Not sure what the logic is here? Does the code require to instantiate a logger now?